lib/player.rb

```diff
@@ -0,0 +1,11 @@
+class Player
+  attr_reader :character
+  attr_reader :name
+  attr_accessor :score
```

Comment:
Use attr_readers/attr_writers instead of attr_accessors

Reply:
Could you tell us why to help us understand?

Reply:
By the way, the draw method is in game.board not in the move(player, num), so the code respect the single responsibility.

Reply:
Using it  allow writing that property from outside the class. That variable can be set to anything

Reply:
That is the purpose, I understand the attribute accessor... I mean why 'use attr_reader/attr_writer instead of attr_accessor?'

Reply:
Kobiyoyo do you mean that we work on attr_accessor :score as attr_reader ? And if yes, could you advise according to the Manuel's question